tests/unit_tests/test_wave_06.py

```diff
@@ -109,16 +117,26 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That result is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
-
-@pytest.mark.skip
+    assert len(tai.inventory) == 3
+    assert item_c not in tai.inventory
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_f in tai.inventory
+    assert len(jesse.inventory) == 3
+    assert item_f not in jesse.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_c in jesse.inventory
```

Comment:
You can combine lines 128 and 133 if you want:
`assert len(tai.inventory) == 3 and len(jesse.inventory) == 3`

You can use list destructing like this too - a pythonic way of doing it:
```python
assert [item_a, item_b, item_f] == tai.inventory
assert [item_d, item_e, item_c] == jesse.inventory
```

https://medium.com/@umaramanat66/destructuring-list-in-python-like-javascript-f7d4c0968538